src/recompilation.cpp

```diff
@@ -440,10 +497,10 @@ bool process_instruction(const RecompPort::Context& context, const RecompPort::C
         print_line("{}{} = hi", ctx_gpr_prefix(rd), rd);
         break;
     case InstrId::cpu_mtlo:
-        print_line("lo = {}{}", ctx_gpr_prefix(rd), rd);
+        print_line("lo = {}{}", ctx_gpr_prefix(rs), rs);
```

Comment:
was this a bug previously?

Reply:
Yup, it was causing the cursor on the pause screen in Zelda recomp to be a slightly wrong color. This instruction is only emitted by clang and it's only used when register pressure is very high, so it didn't come up anywhere else.